### PR TITLE
Update whitelist url

### PR DESCRIPTION
**Pull request by katiedavis**
updating the readme to use the new loopback url

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,13 +49,13 @@ Edit your app’s whitelisted redirection URLs in the Partner Dashboard by navig
...
 https://<LIVE_NGROK_URL>/auth/callback
-http://localhost:3456
+http://app-cli-loopback.shopifyapps.com:3456
 ```
 
 For Rails Apps:
 ```
 https://<LIVE_NGROK_URL>/auth/shopify/callback
-http://localhost:3456
+http://app-cli-loopback.shopifyapps.com:3456
```
Comment (gfscott) [inline README.md]: Is https an option for the loopback?

File: README.md [hunk 2]
```diff
@@ -49,13 +49,13 @@ Edit your app’s whitelisted redirection URLs in the Partner Dashboard by navig
...
 ```
 
 The first URL handles the authentication flow for adding your app to a Shopify development store. The second is currently necessary to allow your local development server to make Shopify API requests on behalf of your development app.
```
Comment (gfscott) [inline README.md]: To me it feels like we should explain in slightly more detail what the loopback is doing and why. It's kind of a mystery URL.

File: README.md (hunk 1 above)
Comment (tanema) [reply README.md]: no

File: README.md (hunk 2 above)
Comment (katiedavis) [reply README.md]: For sure. I'm going to leave that to @tanema (I'll open an issue). I just want to get this in for anyone using the tool currently and running into issues.

**Pull request merged**